config/management/src/json_rpc.rs

```diff
@@ -2,46 +2,75 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::error::Error;
+use crate::TransactionContext;
 use libra_secure_json_rpc::JsonRpcClient;
-use std::{fs, path::PathBuf};
-use structopt::StructOpt;
-
-#[derive(Debug, StructOpt)]
-pub struct SubmitTransaction {
-    #[structopt(long)]
-    pub host: String,
-    #[structopt(long)]
-    pub transaction_path: PathBuf,
+use libra_types::account_config::AccountResource;
+use libra_types::validator_config::ValidatorConfig;
+use libra_types::{
+    account_address::AccountAddress, account_state::AccountState, transaction::SignedTransaction,
+};
+
+/// A wrapper around JSON RPC for error handling
```

Comment:
one could argue that this would use structopt to produce a client

Reply:
We can address this in the future.